### PR TITLE
Pe ordering fix

### PR DESCRIPTION
**Pull request by kgeller**
Just like in https://github.com/elastic/ecs/pull/1603, we realized we were missing `pe.*` under threat.indicator.file and `threat.enrichments.indicator.file' like the [threat RFC](https://github.com/elastic/ecs/blob/master/rfcs/text/0008-threat-intel.md#using-existing-ecs-fields-to-store-indicator-information) describes. I realized this was due to an ordering issue on our field reuses. We solve this issue the same way here for 'group' at 'user'.

Sibling PR to remove the incorrect nestings https://github.com/elastic/ecs/pull/1606

### CODE REVIEW

File: generated/ecs/ecs_nested.yml [hunk 1]
```diff
@@ -14448,6 +14624,13 @@ threat:
   - threat.indicator.x509
   prefix: threat.
   reused_here:
+  - full: threat.indicator.pe
```
Comment (rylnd) [inline generated/ecs/ecs_nested.yml:14627]: I think we want these gone, right? `pe` will be nested under `file` or `process`, but not as a sibling

**Review by rylnd — CHANGES_REQUESTED** [review]
I think we should delete `pe` as a top-level field on `indicator`; otherwise this looks good to me.

**Review by rylnd — APPROVED** [review]
Disregard the last review, it looks like you already made my proposed changes in #1604. :shipit: 

I appreciate the logically separated PRs; I do not envy merging them back together, though!

**Review by epixa — APPROVED** [review]
LGTM

**Pull request merged**